scss/components/tooltip.scss

```diff
@@ -0,0 +1,82 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-tooltip
+    .tn-tooltip__arrow
+    .tn-tooltip__content
+*/
+$block: ns(tooltip);
+
+.#{$block} {
+
+    //LOCAL VARS (set all themeable properties, always include !default)
+    $tn-tooltip-content-background-color: tn-color(text, 3) !default;
+    $tn-tooltip-content-color: tn-color(text-inverse, 1) !default;
+    $tn-tooltip-padding: tn-space(s) !default;
+    $tn-tooltip-arrow-offset: (tn-space(s)) - 2 !default ;
+    $tn-tooltip-arrow-width: $tn-tooltip-arrow-offset !default;
+
+    //BLOCK BASE *******************************************
+    @include reset;
+    background: $tn-tooltip-content-background-color;
+    padding: $tn-tooltip-padding;
+    display: inline-block;
+    position: relative;
+
+    //ELEMENTS *******************************************
+    &__arrow{
+        width: 0;
+        height: 0;
+        border-style: solid;
+        border-width: 0 $tn-tooltip-arrow-width $tn-tooltip-arrow-width $tn-tooltip-arrow-width;
+        border-color: transparent transparent $tn-tooltip-content-background-color transparent;
+        position: absolute;
+        display: none;
+        &--left{
+            @extend .tn-tooltip__arrow;
```

Comment:
These `@extend`s are not necessary since you should include the base class along with the modifier class. See `component.njk` below.